google/cloud/channel/v1/cloud_channel_reports_client.h

```diff
@@ -238,11 +249,14 @@ class CloudChannelReportsServiceClient {
   /// [google.cloud.channel.v1.Row]: @googleapis_reference_link{google/cloud/channel/v1/reports_service.proto#L399}
   ///
   // clang-format on
+  GOOGLE_CLOUD_CPP_DEPRECATED("This RPC is deprecated.")
   StreamRange<google::cloud::channel::v1::Row> FetchReportResults(
       std::string const& report_job, Options opts = {});
 
   // clang-format off
   ///
+  /// @deprecated This RPC is deprecated.
```

Comment:
I am not sure what it looks like when the `@deprecated` tag shows up before the brief description (the first line of the comment block). Can you check?

There is a build that genreates the documentation:
```
ci/cloudbuild/build.sh -t publish-docs-pr
```

Then you can find the HTML in the build-out directory and open it in a browser. e.g:
`build-out/fedora-latest-publish-docs/publish-docs/__default__/cmake-out/google/cloud/channel/html/classgoogle_1_1cloud_1_1channel__v1_1_1CloudChannelReportsServiceClient.html`

(or open the `google/cloud/channel/html/index.html` and navigate the site)

Reply:
It looks like https://screenshot.googleplex.com/7RrkgQTsXkPqpEX

Reply:
Cool. Thanks